apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -2070,9 +2070,30 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
   test "not spent or not existed utxo should be not challengeable", %{
     processor_filled: processor
   } do
-    assert {:ok, 1000, exit_info} = Core.get_challange_data({:ok, 1000}, @utxo_pos1, processor)
+    assert {:ok, 1000, exit_info} = Core.get_challenge_data({:ok, 1000}, @utxo_pos1, processor)
 
-    assert {:error, :utxo_not_spent} = Core.get_challange_data({:ok, :not_found}, Utxo.position(1000, 0, 1), processor)
-    assert {:error, :exit_not_found} = Core.get_challange_data({:ok, 1000}, @utxo_pos3, processor)
+    assert {:error, :utxo_not_spent} = Core.get_challenge_data({:ok, :not_found}, @utxo_pos2, processor)
+    assert {:error, :exit_not_found} = Core.get_challenge_data({:ok, 1000}, @utxo_pos3, processor)
+  end
+
+  @tag fixtures: [:processor_filled, :alice, :transactions]
+  test "challenges standard exits challengeable by IFE transactions",
+       %{processor_filled: processor, alice: alice, transactions: [ife_tx | _]} do
```

Comment:
maybe will be better capture alice_addres instead of whole alice.
```suggestion
       %{processor_filled: processor, alice: %{addr: alice_addr}, transactions: [ife_tx | _]} do
```